packages/twenty-server/src/workspace/workspace-query-runner/workspace-pre-query-hook/workspace-pre-query-hook.config.ts

```diff
@@ -0,0 +1,11 @@
+import { MessageFindManyPreQueryHook } from 'src/workspace/messaging/query-hooks/message/message-find-many.pre-query.hook';
+import { MessageFindOnePreQueryHook } from 'src/workspace/messaging/query-hooks/message/message-find-one.pre-query-hook';
+import { WorkspaceQueryHook } from 'src/workspace/workspace-query-runner/workspace-pre-query-hook/types/workspace-query-hook.type';
+
+// TODO: move to a decorator
```

Comment:
Hmm I think it will be better in a decorator, but can we do it actually ?
You think on something like that:

```typescript
@WorkspaceAddHook({
  type: WorkspaceHookType.Prehook,
  on: [
    {
       entity: 'message',
       resolvers: ['findOne', 'findMany'],
    }
  ],
})
```

Reply:
Yes 🤔 Not sure yet that's why I postponed this for now